eng/pipelines/templates/stages/cosmos-sdk-client.yml

```diff
@@ -22,36 +19,38 @@ extends:
   template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: ${{ parameters.ServiceDirectory }}
-    # Skip emulator tests in the CI run
     TestMarkArgument: not cosmosEmulator
     InjectedPackages: ${{parameters.InjectedPackages}}
     Artifacts: ${{parameters.Artifacts}}
 
-#  - stage: Test_Emulator
-#    dependsOn: []
-#    jobs:
-#    - job: Emulator
-#      strategy:
-#        matrix:
-#          Windows_Python36:
-#            OSVmImage: 'windows-2022'
-#            PythonVersion: '3.6'
-#      pool:
-#        vmImage: $(OSVmImage)
-#
-#      steps:
-#        - template: /eng/common/pipelines/templates/steps/cosmos-emulator.yml
-#          parameters:
-#            EmulatorMsiUrl: ${{ parameters.EmulatorMsiUrl }}
-#            StartParameters: ${{ parameters.EmulatorStartParameters }}
-#
-#        - template: /eng/pipelines/templates/steps/build-test.yml
-#          parameters:
-#            TestMarkArgument: not globaldb
-#            EnvVars:
-#              ACCOUNT_HOST: https://localhost:8081/
-#            ServiceDirectory: ${{ parameters.ServiceDirectory }}
-#            PythonVersion: $(PythonVersion)
-#            OSVmImage: $(OSVmImage)
-#            ToxTestEnv: 'whl,sdist'
-#            InjectedPackages: ${{parameters.InjectedPackages}}
+stages:
+  - stage: Test_Emulator
+    displayName: Test Emulator
+    dependsOn: []
+    jobs:
+    - job: Emulator
+      strategy:
+        matrix:
+          Windows_Python36:
+            OSVmImage: 'windows-2022'
+            PythonVersion: '3.6'
+      pool:
+        vmImage: $(OSVmImage)
+
+      steps:
+        - template: /eng/common/pipelines/templates/steps/cosmos-emulator.yml
+          parameters:
+            EmulatorMsiUrl: ${{ parameters.EmulatorMsiUrl }}
+            StartParameters: ${{ parameters.StartParameters }}
+
+        - template: /eng/pipelines/templates/steps/build-test.yml
+          parameters:
+            TestMarkArgument: not globaldb
+            EnvVars:
+              ACCOUNT_HOST: 'https://localhost:8081/'
```

Comment:
You might need to set a couple more so that they run in "live" mode against the local cosmos emulator right? Playback tests aren't exercising the emulator if I'm understanding what this does correctly.